### ISSUE TITLE
EVPN multihoming lacp port-id range support

### ISSUE THREAD

**Issue by dgonzalez-arista** [aristanetworks/ansible-avd#1074]
**Issue Type**

- [x] Template enhancement

Add support to configure global command "lacp port-id range", which is part of the recommended configuration for EVPN multihoming:
https://www.arista.com/custom_data/downloads/?f=/support/download/DesignGuides/EVPN_Multihoming_Design_Guide.pdf

"A global configuration command must be used to configure LACP port-id range for all PEs connected to multihomed CE(s).
When the port-id range command is configured, it limits the range of allocated port-ids to the one specified.

If the CE is receiving the packet from two different ports in the same LAG (one each to a MH PE) with the same port-id, the CE device interprets (wrongly) that the port has moved from one LAG to a different LAG. Hence, configure the range for the port-id on the PE router to ensure there are no overlapping ranges for the PEs in the same ES."

CLi configuration:
```
lacp port-id range <min_value> <max_value> (1-64K)
```
Similar to RFE: #1043 

**Describe the solution you'd like**
Ideally when multihoming is configured in the SERVER.yml file, also ensure separate port-id ranges are allocated to the PEs in that same ES.
 
For example, multihoming peer A (assuming two peers, and up to 127 port-channels):
```
lacp port-id range 1 127
```
Multihoming peer B:
```
lacp port-id range 128 255
```
These range could overlap in other vteps, same range applied to MH pair Leaf1/2,Leaf3/4, etc.

thanks,
David

**Comment by ClausHolbechArista** [comment]
Since this is a global command for the switch, we should not have it under the endpoint data model. We could add a knob to the fabric-topology data model, so it can be set differently per leaf as needed.

**Comment by tgodaA** [comment]
Yeah, but the issue #1295 is to support global lacp parameters. Then we can add it to eos_designs l3leaf to support #1074.

Does it sound good?

**Comment by tgodaA** [comment]
Issue solved in PR#1301

**Comment by dgonzalez-arista** [comment]
Tested and works fine. As of now this needs can be set on each leaf in a host_var file:

```
custom_structured_configuration_lacp:
  port_id:
    range:
      begin: 1
      end: 127
  rate_limit:
    default: true
```

So different values can be selected for peer A and peer B. Ideally this could be done automatically for all topology leaves as discussed earlier.

thanks!
